src/lib/utils.ts

```diff
@@ -0,0 +1,6 @@
+import { type ClassValue, clsx } from "clsx"
+import { twMerge } from "tailwind-merge"
```

Comment:
next time when adding a big library like shadcn, let's split it up into two prs so they are more distinct / easier to review. 

best to have Prs be distinct cuts, easier to revert that way and way easier to review